### DISCUSSION TITLE
HIghest Recall Trainings

### DISCUSSION THREAD

**Discussion by jquaglia-uss** [ultralytics/ultralytics#7993] [Q&A]
Ultralytics Community,

Is there anyway I can tweak settings etc in order to have my model most heavily favor Recall?   My use case is very dependent on not missing detections.  I have devised some downstream ways to clean up False Detections and preserve my accuracy overall, but it's critical for me to have the first stage detector be as sensitive and accurate as possible.

Does anyone have any thoughts or ideas on how to do this?

**Comment by pderrenger** [comment]
@jquaglia-uss hello!

To prioritize recall during training with YOLOv8, you can adjust the loss function weights to emphasize detection over precision. Specifically, you can increase the weight of the objectness loss and decrease the weight of the class loss. This will encourage the model to focus on detecting objects, even at the risk of increasing false positives, which you mentioned you can handle downstream.

Additionally, you can experiment with the confidence threshold during inference. Lowering this threshold will result in more detections being considered, again potentially increasing recall at the expense of precision.

Remember to monitor your validation metrics closely to ensure that the adjustments are having the desired effect on recall without excessively compromising overall model performance.

For more detailed guidance on these settings, please refer to our documentation on training and validation modes. Good luck with your sensitive detection tasks! 🚀
